### DISCUSSION TITLE
ScriptReference/Mathf.RoundToInt

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#1] [Notes]
# ScriptReference/Mathf.RoundToInt

https://docs.unity3d.com/ScriptReference/Mathf.RoundToInt.html

**Comment by vertxxyz** [comment]
If you want to always round up from `0.5` increments, use the [`System.Math`](https://docs.microsoft.com/en-us/dotnet/api/system.math.round) class:
```csharp
(int) Math.Round(value, MidpointRounding.AwayFromZero)
```
